commands/parsePackageJson.go

```diff
@@ -84,3 +86,42 @@ func ParseSectionConfig(sectionConfigContents string) (SectionConfigJSON, error)
 	}
 	return sectionConfig, nil
 }
+
+// JSON returns a Resolver that retrieves values from a JSON source.
+//
+// Hyphens in flag names are replaced with underscores.
+func PackageJSONResolver(r io.Reader) (kong.Resolver, error) {
+	buf := new(bytes.Buffer)
+	_, err := buf.ReadFrom(r)
+	if err != nil{
+		return nil,nil
```

Comment:
We should return errors so they can be handled, or at least handle it by logging if that's what we want to happen.

Reply:
this is kong internals
it's basically https://github.com/alecthomas/kong/blob/master/resolver.go#L33

If it can't read the package json, then it will get the flags from the user input